koans/ArrayKoans.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace PhpKoans;
+
+use PHPUnit\Framework\TestCase;
+
+defined('__') or define('__', null);
+
+class ArrayKoans extends TestCase
+{
+    /**
+     * @testdox
+     */
+    public function getTheValueOfTheFirstElementInAnArray()
```

Comment:
if we are using the test... on the format of the tests, lets stick to that.
I thing that we should talk about this